tests/cypress/integration/publisher/000-general/03-create-and-publish-graphql-api-with-all-information.spec.js

```diff
@@ -22,8 +22,8 @@ describe("Create GraphQl API from file", () => {
     const filepath = 'api_artifacts/schema_graphql.graphql';
     const modifiedFilepath = 'api_artifacts/modified_schema_graphql.graphql';
     const apiVersion='1.0.0';
-    const apiContext="/swapi";
-    const apiName='StarWarsAPIGQL';
+    const apiContext=`/swapi${Math.floor(Math.random() * (100000 - 1 + 1) + 1)}`;
```

Comment:
If we can use a Util function to get this random number generated.

Reply:
Create a util method